static/src/inline-svgs/commercial/logo-membership.svg

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This SVG seems to have a non-standard 'inkspace' namespace on several elements. The inline SVG processor, using SVGO, can't handle them. You can check your file against the [W3 validator](https://validator.w3.org/check). You might have more luck using Inkscape's "Export to Plain SVG" functionality.